### PR TITLE
[ADD][12.0] new module - account_invoice_tax_note_view

### PR DESCRIPTION
**Pull request by badbole**
from module descritpion:

Two OCA modules are present and both depends on account,

1. account_invoice_tax_note - adding a tax group note on account.tax.group model
   - but does not add a field to view, because it depends on account module,
     which by default has no account.tax.group views defined
   - default odoo form and tree views are generated "on the fly" and added field
     "note" is visible and editale on form view.

2. account_group_menu - adds missing menuitems and defines views for account tax group
   but is not aware of any extensions or added fields.

This module is connector between those two and adds a field to form view so
users won't be confused with field not visible after module instalation.

### CODE REVIEW

**Comment by badbole** [comment]
@rousseldenis @acsonefho could you please check this pull request , as is extends your original module account_tax_invoice_note , also please suggest if it does not belog here.. tnx

**Review by acsonefho — APPROVED** [review]
LGTM (code only)

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
Hi, could you check conflicts?

File: account_invoice_tax_note_view/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Account invoice tax note view",
+    'version': "12.0.1.0.0",
+    "author": "Decodio Applications,Odoo Community Association (OCA)",
+    "development_status": "Production/Stable",
+    "summary": "Adding missing filed on view when OCA account_invoice_tax_note """
```
Comment (sergiocorato) [inline account_invoice_tax_note_view/__manifest__.py]: Mixed `'` and `"`, three `"` at the end of line why?

**Pull request closed without merging**